dyno-demo/src/main/java/com/netflix/dyno/demo/redis/DynoJedisDemo.java

```diff
@@ -562,36 +565,39 @@ private String constructRandomValue(int sizeInKB) {
 		return ss;
 	}
 	private List<Host> getHostsFromDiscovery(final String clusterName) {
-		
-		String url = "http://discovery.cloudqa.netflix.net:7001/discovery/v2/apps/" + clusterName;
+
+		String env = System.getProperty("netflix.environment", "test");
+		String discoveryKey = String.format("dyno.demo.discovery.%s", env);
+
+		if (!System.getProperties().containsKey(discoveryKey)) {
+			throw new IllegalArgumentException("Discovery URL not found");
+		}
+
+		final String url = String.format("http://%s/%s", System.getProperty(discoveryKey), clusterName);
```

Comment:
Interesting, you are constructing the URL based on the properties... Looks good!